ops/services/alerts/app_service_metrics/database.tf

```diff
@@ -22,7 +22,8 @@ ${local.skip_on_weekends}
   }
 
   action {
-    action_group = var.action_group_ids
+    action_group           = var.action_group_ids
+    custom_webhook_payload = "{\"Alert Response Docs\": \"https://github.com/CDCgov/prime-simplereport/wiki/Alert-Response\" }"
```

Comment:
is this value, `custom_webhook_payload`, something we can store as a variable 🤔?

Reply:
I went ahead and updated it based on your suggestion!